### DISCUSSION TITLE
Support Workflow Commands

### DISCUSSION THREAD

**Discussion by nikeee** [rome/tools#4575] [Suggestions]
GitHub actions supports putting annotations in source files during a CI run:
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

Feature request: `rome ci` should detect if it is running in GitHub actions and emit error/warning messages for linter and style issues.

eslint seems to support this as well. It looks like this (for `no-unused-vars`):
![image](https://github.com/rome/tools/assets/4068864/36433c5e-0bf2-41eb-bacb-860a66e5cbcf)

If there is a reliable way to detect if rome is running in GH Actions[^1], this should be possible to provide without breaking anything. Maybe only consider doing it when using `rome ci`. Eslint seems to do this by default, if it is running in GHA.

[^1]: Should be possible when checking env var [`GITHUB_ACTIONS` for `true`](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables).

**Comment by ematipico** [comment]
This is a feature that is definitely in our radars! The old Rome code base had a feature like this.

We're **definitely** open to support it, we just lack the manpower to do so at the moment. So if someone wants to do the heavy lifting, we're definitely willing to help!
